admin_guide/cluster_management.rst

```diff
@@ -133,8 +133,10 @@ The new shard(s) are created on the same node as the shard(s) from which the ten
 
   -- find the node currently holding the new shard
   SELECT nodename, nodeport
-    FROM pg_dist_shard_placement
-   WHERE shardid = 102240;
+    FROM pg_dist_placement AS placement,
+         pg_dist_node AS node
+   WHERE placement.groupid = node.groupid
```

Comment:
Should we also maybe query for where noderole is 'primary'? In a cluster with followers, this will return multiple rows.